### ISSUE TITLE
Microsoft Visual C++ 2015 Redistributable - 64 bit or 32 bit

### ISSUE THREAD

**Issue by jawn** [aspnet/AspNetCore.Docs#14788]
I assume I can take the 64 bit if the system supports it - correct?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f92a99a8-6ffa-e5f9-b7ef-f13fa1e0f17b
* Version Independent ID: 51273790-5178-875b-010e-15d7725da535
* Content: [Publish an ASP.NET Core app to IIS](https://docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-iis?view=aspnetcore-3.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/publish-to-iis.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/publish-to-iis.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
@joeloff, do you know why the hosting bundle installer requires the Microsoft Visual C++ 2015 Redistributable? We were discussing this on another issue yesterday, but engineering didn't know why the installer was installing it. How does the hosting bundle installer handle this? Does it install both x86 and x64?

**Comment by joeloff** [comment]
This was only part of the 1.0/1.1 hosting bundles. The .NET Core runtime bundle that we shipped back then downloaded and installed the C++ runtime if it wasn't installed on the machine. We listed it explicitly because enterprise customers in locked down environments ran into problem when the servers didn't have outside internet connections so their admins had to manually download and copy the c++ redists to the servers, install it and only then could they install the hosting bundle.
